ocamldoc/odoc_messages.ml

```diff
@@ -388,7 +388,7 @@ let authors = "Author(s)"
 let version = "Version"
 let since = "Since"
 let before = "Before"
-let deprecated = "Deprecated."
+let deprecated = "Deprecated. "
```

Comment:
This adds unwanted space in a few places, e.g.:
https://github.com/ocaml/ocaml/blob/59320b5c08f4879344a901e4c4c2a4888b551e39/ocamldoc/odoc_texi.ml#L532
and
https://github.com/ocaml/ocaml/blob/trunk/ocamldoc/odoc_man.ml#L209-L210
I think the fix needs to be here instead:
https://github.com/ocaml/ocaml/blob/59320b5c08f4879344a901e4c4c2a4888b551e39/ocamldoc/odoc_html.ml#L746-L747
Incidentally, this usage looks wrong (generating `Deprecated.!`, but I didn't dig further as to where:
https://github.com/ocaml/ocaml/blob/59320b5c08f4879344a901e4c4c2a4888b551e39/ocamldoc/odoc_misc.ml#L220

Reply:
I have changed the message to "Deprecated" and updated other places to add the full stop and/or space. I have checked the HTML, text, and PDF output and they are now correct.